### ISSUE TITLE
Ninjas get stuck draining if they attempt to drain an empty smes

### ISSUE THREAD

**Issue by Iamgoofball** [NTStation/NTstation13#143]
_Issue by **[KazeEspada](https://github.com/KazeEspada)** from Tuesday Sep 17, 2013 at 23:37 GMT_
_Originally opened as https://github.com/tgstation/-tg-station/issues/1392_

---

INB4: FUCKING NINJAS.
This is probably just a missing sanity check to make sure the target power thing has power in it.

It was witnessed on Phillip. Revision is before click standardization(because that commit has a big change on ninja gloves).

Basically little suzy the ninja tried to drain an empty smes and got her draining var stuck on 1. Well, everytime a ninja trys to drain, it makes sure its not already draining. Poor suzy couldn't sap power anymore until a nice person with var edit powers fixed the draining var.

**Comment by Iamgoofball** [comment]
_Comment by **[Cheridan](https://github.com/Cheridan)** from Friday Sep 20, 2013 at 20:30 GMT_

---

FUCKING NINJAS.
